Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -1320,309 +1319,29 @@ def add_note_to_asset_command(client: Client, args: dict[str, Any]) -> CommandRe
     return command_results
 
 
-def ip_command(client: Client, args: dict[str, Any]) -> list[CommandResults]:
-    """
-    ip command returns enrichment for an IP address.
```

Comment:
@BigEasyJ @johnnywilkes Is there agreement to deprecate the commands this way? Usually we keep the functionality of the command as is. I would be happy to confirm with you and understand why it was done this way.